.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-max-line-length = 80
+max-line-length = 88
```

Comment:
Isn't there a command that ignores this line check? So that we won't have to edit .flake8 itslef

Reply:
In the latest OCA repository template, they [ignore regular line length warnings](https://github.com/OCA/oca-addons-repo-template/blob/master/src/.flake8#L8) and instead replace by [bugbear line warnings of max-line-length 88](https://github.com/OCA/oca-addons-repo-template/blob/master/src/.flake8#L2). Bugbear looks at the (rolling?) average, so that if one line goes over 88 to like 92 but the rest of the lines are OK, it will tolerate that one line.

Reply:
So this is just an application of the latest template really.